### DISCUSSION TITLE
Ability to allowlist IPs on dashboard

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#7427] [Feature Requests]
*This discussion has been created following the [thread on the Discord](https://discord.com/channels/839993398554656828/989565877252931595)*

Hi 👋🏻 
Would it be possible to add the possibility to allowlist our IPs on the dashboard, to avoid the problem with ``fail2ban``?

Thank you!

**Comment by saltcod** [comment]
Is this one covered by Network Restrictions?
https://supabase.com/dashboard/project/_/settings/database#network-restrictions

**Reply by Olyno** [reply]
Hi! This is probably the way! Thank you :p
